src/lib/set-up-event-handlers.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+function setUpEventHandlers(element, func) {
```

Comment:
I'd like to rename this to something more descriptive, but I'm coming up with a blank. `setUpSelectionEventHandlers`? `addSelectionEventHandler`?